community/contributingGuides/Contributing.asciidoc

```diff
@@ -15,15 +15,15 @@ As an open source project we are always on the lookout for community members tha
 
 === Apache License
 
-All contributions to AeroGear are made under the http://apache.org/licenses/LICENSE-2.0.txt[Apache License, Version 2.0].
+All contributions to AeroGear are made under the http://apache.org/licenses/LICENSE-2.0.txt[Apache License, Version 2.0^].
```

Comment:
Is the `^` intentional? What does it mean?

Reply:
@pb82 it opens a new tab. I used it anywhere there is a link to something external from the site.

Reply:
Thanks, didn't know that